### PR TITLE
<Card/> - collapsed Card should not have any bottom divide

### PR DESCRIPTION
**Pull request by dbarabashdev**
What do you think about this solution?

### CODE REVIEW

File: src/Card/CollapsedHeader/CollapsedHeader.js [hunk 1]
```diff
@@ -78,6 +78,11 @@ class CollapsedHeader extends WixComponent {
       [styles.withDivider]: !withoutDivider
     });
 
+    const headerClassesWithoutDivider= classNames({
```
Comment (argshook) [inline src/Card/CollapsedHeader/CollapsedHeader.js]: space before `=`

File: src/Card/CollapsedHeader/CollapsedHeader.js [hunk 2]
```diff
@@ -78,6 +78,11 @@ class CollapsedHeader extends WixComponent {
...
+      [styles.headerOnlyTitle]: !subtitle,
+      [styles.headerTitleSubtitle]: subtitle,
```
Comment (argshook) [inline src/Card/CollapsedHeader/CollapsedHeader.js]: unnecessary comma

File: src/Card/CollapsedHeader/CollapsedHeader.js [hunk 3]
```diff
@@ -116,10 +121,11 @@ class CollapsedHeader extends WixComponent {
     ) : null;
 
     const toggleElement = this.props.toggleStyle === 'switch' ? switchElement : buttonElement;
+    const switchHeader = this.state.isCollapsed ? headerClassesWithoutDivider : headerClasses
 
     return (
       <div>
-        <div className={headerClasses} onClick={this.onToggleChange}>
+        <div className={switchHeader} onClick={this.onToggleChange}>
```
Comment (argshook) [inline src/Card/CollapsedHeader/CollapsedHeader.js]: i'd just put condition right in `className={here}`, but it doesn't really matter

**Comment by dbarabashdev** [comment]
Fixed all `request changes`

**Comment by argshook** [comment]
Great, thank you :)

**Pull request merged**